### PR TITLE
Fixing autoregister and adding repositories

### PR DESCRIPTION
**Pull request by IvanShamatov**
I experiment with relations/repositories. Also autoregistering has been fixed.

### CODE REVIEW

File: lib/repositories/talk_repo.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+module Repositories
```
Comment (k0va1) [inline lib/repositories/talk_repo.rb:1]: 👍

File: lib/commands/beers.rb [hunk 2]
```diff
@@ -1,5 +1,3 @@
-require_relative 'base'
```
Comment (k0va1) [inline lib/commands/beers.rb:1]: why we don't need it now?

File: system/boot.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
+begin
+  require "pry-byebug"
```
Comment (k0va1) [inline system/boot.rb]: i think single quotes is more preferred for strings without interpolation

Comment (k0va1) [reply system/boot.rb]: btw, it will be great to require `pry-byebug` only in development and test environments, WDYT?

File: system/boot/rom.rb [hunk 4]
```diff
@@ -1,16 +1,13 @@
...
+    require 'rom-repository'
   end
 
   start do
-    configuration = ROM::Configuration.new(:yaml, 'data')
-    configuration.register_relation(Relations::Jobs)
-    configuration.register_relation(Relations::Talks)
-    # configuration.auto_registration('lib')
+    configuration = ROM::Configuration.new(:yaml, app.root.join('data'))
+    configuration.auto_registration(app.root.join('lib'))
```
Comment (IvanShamatov) [inline system/boot/rom.rb]:
```
shamatov@laptop ~/Projects/GranatProject/saintpruby_bot (boot-file-to-load-app●)$ irb                                                                                                         [ruby-2.6.3]
2.6.3 :001 > require 'bundler'; Bundler.require ; require_relative 'system/boot'
Traceback (most recent call last):
       16: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/dry-system-0.12.0/lib/dry/system/booter.rb:86:in `block (2 levels) in init'
       15: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/dry-system-0.12.0/lib/dry/system/booter.rb:99:in `block (2 levels) in start'
       14: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/dry-system-0.12.0/lib/dry/system/components/bootable.rb:97:in `start'
       13: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/dry-system-0.12.0/lib/dry/system/lifecycle.rb:54:in `call'
       12: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/dry-system-0.12.0/lib/dry/system/lifecycle.rb:54:in `each'
       11: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/dry-system-0.12.0/lib/dry/system/lifecycle.rb:56:in `block in call'
       10: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/dry-system-0.12.0/lib/dry/system/lifecycle.rb:84:in `start'
        9: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/dry-system-0.12.0/lib/dry/system/lifecycle.rb:119:in `trigger!'
        8: from /Users/shamatov/Projects/GranatProject/saintpruby_bot/system/boot/rom.rb:9:in `block (2 levels) in <top (required)>'
        7: from /Users/shamatov/.rvm/rubies/ruby-2.6.3/lib/ruby/2.6.0/forwardable.rb:230:in `auto_registration'
        6: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/rom-core-5.0.2/lib/rom/setup.rb:51:in `auto_registration'
        5: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/rom-core-5.0.2/lib/rom/setup/auto_registration.rb:58:in `relations'
        4: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/rom-core-5.0.2/lib/rom/setup/auto_registration.rb:81:in `load_entities'
        3: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/rom-core-5.0.2/lib/rom/setup/auto_registration.rb:81:in `map'
        2: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/rom-core-5.0.2/lib/rom/setup/auto_registration.rb:99:in `block in load_entities'
        1: from /Users/shamatov/.rvm/gems/ruby-2.6.3/gems/dry-inflector-0.1.2/lib/dry/inflector.rb:78:in `constantize'
NameError (uninitialized constant Lib)
Did you mean?  Zlib
```

**Pull request merged**